src/support/AliasFor.ts

```diff
@@ -1,20 +0,0 @@
-import { Entity } from '../Entity';
-
-export function AliasFor(key?: string) {
```

Comment:
Why was this removed? If there's a good reason, please explain that change in the PR description, so it can be used for release notes.

Reply:
It's baked in to `@Type()` decorator. I thought there is very little reason to use this one other than AliasFor being more semantically reasonable.

Reply:
I have added extra details in the PR description.